prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -243,10 +243,14 @@ public override Task<bool> ProcessRule(Enrollee enrollee)
 
     public class IsPotentialPaperEnrolleeReturnee : AutomaticAdjudicationRule
     {
+        private readonly IBusinessEventService _businessEventService;
         private readonly IEnrolleePaperSubmissionService _enrolleePaperSubmissionService;
 
-        public IsPotentialPaperEnrolleeReturnee(IEnrolleePaperSubmissionService enrolleePaperSubmissionService)
+        public IsPotentialPaperEnrolleeReturnee(
+            IBusinessEventService businessEventService,
+            IEnrolleePaperSubmissionService enrolleePaperSubmissionService)
```

Comment:
Same as other comment:
You're passing the constructor's positional arguments arguments in the wrong order
Passing: new IsPotentialPaperEnrolleeReturnee(_enrolleePaperSubmissionService, _businessEventService)
Constructor: public IsPotentialPaperEnrolleeReturnee(IBusinessEventService businessEventService, IEnrolleePaperSubmissionService enrolleePaperSubmissionService)